src/runtime/modality-handler.ts

```diff
@@ -16,7 +16,7 @@ import {SlotDomConsumer} from './slot-dom-consumer.js';
 
 export class ModalityHandler {
   constructor(public readonly slotConsumerClass: typeof SlotConsumer,
-              public readonly descriptionFormatter?: typeof DescriptionFormatter) {}
+              public descriptionFormatter?: typeof DescriptionFormatter) {}
```

Comment:
just curios, what is the test that overrides it?

Reply:
It's in:

```
src/tests/recipe-descriptions-test.ts:88:    helper.arc.pec.slotComposer.modalityHandler.descriptionFormatter = options.formatter;
```